database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -136,21 +136,7 @@ abstract class SlickDatabase(override val originalDatabaseConfig: Config) extend
 
   protected[this] lazy val insertBatchSize = databaseConfig.getOrElse("insert-batch-size", 2000)
 
-  /*
-   * If you're about to (re-)introduce slick upserts (or insertOrUpdates):
-   *  See https://github.com/broadinstitute/cromwell/pull/5332 which removed them (due to failing tests in Slick 3.3.2)
-   *  Make sure the new slick version you're using passes KeyValueDatabaseSpec (and all the others, of course)
-   *  The problem with Slick upserts was that for MySql it generates wrong sql code:
-   *    `insert ignore into table() values() on duplicate key update ...`
-   *  which makes MySql to ignore any errors which may occur during statement execution. The proper sql code should be
-   *  without `ignore`.
-   *  https://github.com/slick/slick/issues/2045
-   *  https://github.com/slick/slick/issues/2076
-   *
-   * Note: Before the removal, this line used to be:
-   *  = dataAccess.driver.capabilities.contains(JdbcCapabilities.insertOrUpdate)
-   */
-  protected[this] lazy val useSlickUpserts = false
+  protected[this] lazy val useSlickUpserts = dataAccess.driver.capabilities.contains(JdbcCapabilities.insertOrUpdate)
```

Comment:
Can we avoid this check altogether? Right now postgres is not using slick upserts and I wonder whether than means that it suffers from the same deadlock issue as mysql did (and we're only not seeing it because we only have a mysql deadlock test?)